react/components/AddressForm.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { Fragment, PureComponent } from 'react'
```

Comment:
how about using a different name so not to confuse with https://github.com/vtex/address-form/? how about `AddressInput`?